pkg/util/const.go

```diff
@@ -0,0 +1,26 @@
+package util
+
+import "time"
+
+const (
+	// Operand and operator run in the same namespace
+	OperatorNamespace = "openshift-cluster-csi-drivers"
+	OperatorName      = "ibm-vpc-block-csi-driver-operator"
+	OperandName       = "ibm-vpc-block-csi-driver"
+
+	InstanceName = "vpc.block.csi.ibm.io"
+	Resync       = 20 * time.Minute
```

Comment:
If `sync` returns an error, it is called again with exp. backoff, where the first step is really short (~1 second?). Only when `sync` returns `nil` *and* all the informers do not get any event (= the input Secret or ConfigMap did not change), only then is `sync` called after 20 minutes.

In ideal case, when a controller has informer on all its inputs, the periodic `sync` should not be necessary. For example Kubernetes controller-manager has resync every 12 hours. I think that this controller *has* informers on all its inputs, so 20 minutes is a good value.

(Some controllers in library-go have 1 minute resync. At least for some of them this is necessary, because they don't have necessary informers. But for the others it's just waste of CPU).

Reply:
Got it, thanks Jan.